easybuild/easyconfigs/g/gretl/gretl-2020a-foss-2019a.eb

```
@@ -0,0 +1,40 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Loris Bennett
+# ZEDAT, Freie Universitaet Berlin
+
+easyblock = 'ConfigureMake'
+
+name = 'gretl'
+version = '2020a'
+
+homepage = 'http://gretl.sourceforge.net'
+description = "A cross-platform software package for econometric analysis"
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+toolchainopts = {'openmp': True}
+
+# http://prdownloads.sourceforge.net/gretl/gretl-2020a.tar.xz
+source_urls = ['http://prdownloads.sourceforge.net/%(name)s']
+sources = [SOURCELOWER_TAR_XZ]
+checksums = ['4c369f33c9aa825920229263bfd12560168c4e46282240bee3497436a7ae9b7b']
+
+builddependencies = [
+    ('gnuplot', '5.2.6'),
+    ('libxml2', '2.9.8'),
+    ('GMP', '6.1.2'),
+    ('GtkSourceView', '3.24.11'),
+    ('ImageMagick', '7.0.8-46'),
+    ('MPFR', '4.0.2'),
+]
```

My test failed due to a missing dependency on `cURL`
```suggestion
    ('cURL', '7.63.0'),
]
```